tests/src/behaviors/index.ts

```diff
@@ -41,40 +41,25 @@ if (trial === "gossip") {
         }
     })
 } else if (trial === "signal") {
-    const period = 20 * 1000
+    const period = 20 * 1000  // timeout
     orchestrator.registerScenario('Measuring messages per-second--signals', async (s, t) => {
-        let txCount = 4
-        let txAtMax
+        let txCount = 100
+        let duration
+        let txPerSecondAtMax = 0
+        t.comment(`trial with ${config.nodes} nodes, ${config.conductors} conductors per node and ${config.instances} cells per conductor`)
         do {
-            t.comment(`trial with ${config.conductors} conductors per node and ${config.instances} cells per conductor`)
-            txAtMax = 0
-            let actual
-            let txPerSecondAtMax = 0
-            do {
-                t.comment(`trial with ${txCount} tx per ${period}ms`)
-                // bump the scenario UUID for each run of the trial so a different DNA hash will be generated
-                s._uuid = uuidv4();
-                actual = await signalTx(s, t, config, period, txCount, local)
-                const txPerSecond = actual / period * 1000
-                if (txPerSecond > txPerSecondAtMax) {
-                    txAtMax = txCount
-                    txPerSecondAtMax = txPerSecond
-                }
-                if (actual < txCount) {
-                    t.comment(`failed when attempting ${txCount} messages`)
-                    txCount /= 2
-                } else {
-                    t.comment(`succeeded when attempting ${txCount} messages`)
-                    txCount *= 2
-                }
-            } while (txCount > txAtMax)
-
-            t.comment(`test context: ${config.nodes * config.conductors} total conductors`)
-            t.comment(`maxed message per second when sending ${txAtMax}: ${txPerSecondAtMax.toFixed(1)} (sent over ${period / 1000}s)`)
-
-            config.conductors += 1
-            config.instances += 1
-        } while (txAtMax !== 4)
```

Comment:
Why not preserve my logic for increasing conductor count gradually?

Reply:
I was just trying to simplify for the exact need of the test.  We can add back in.

Reply:
Fair enough. Let's skip it